docs/topics/images.rst

```diff
@@ -201,7 +201,7 @@ Therefore, if you'd added the field ``author`` to your AbstractImage in the abov
 
 
 .. Note::
-    The image property used for the ``src`` attribute is actually ``image.url``, not ``image.src``.
+    The image property used for the ``src`` attribute is actually ``image.full_url``, not ``image.src``.
```

Comment:
The `full_url` property isn't intended to replace `url` - it's provided as an alternative, and `url` will still be preferred for images shown on page - so it'd better to add a new section covering `full_url`.